src/main/java/org/apache/cassandra/sidecar/config/yaml/VertxMetricsConfigurationImpl.java

```diff
@@ -32,34 +29,28 @@ public class VertxMetricsConfigurationImpl implements VertxMetricsConfiguration
     public static final boolean DEFAULT_ENABLED = true;
     public static final boolean DEFAULT_EXPOSE_VIA_JMX = false;
     public static final String DEFAULT_JMX_DOMAIN_NAME = "sidecar.vertx.jmx_domain";
-    public static final List<String> DEFAULT_MONITORED_SERVER_ROUTE_REGEXES = Collections.singletonList("/api/v1/.*");
```

Comment:
where will this go?

Reply:
We decided to not use this. Instead use the include and exclude pattern we provide for filtering.

Reply:
should we add it in  `sidecar.yaml` then?

Reply:
Discussed with Fracisco, we will use the exclude list instead of configuring monitored server routes